src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Dockerfile

```diff
@@ -1,18 +1,15 @@
-FROM microsoft/mssql-server-linux:latest
+FROM mcr.microsoft.com/mssql/server:2019-latest
```

Comment:
Any reason for the new image? Is this the version we are using in Azure SQL Server?

Reply:
The one that we were using was MSSQL Server 2017, even though our sqlproj. targets 2019/Azure SQL. We encountered this with @bjss-chris when we tried to move from using custom scripts for deployment to using dacpac. I assumed it would be beneficial for us to upgrade to the newest version. 

This would probably mean to revisit isapi-db and apply similar changes to its Dockerfile.

Thoughts @sam-roberts-bjss ?

Reply:
As long as it is compatible with the version of Azure SQL Server I don't mind & yes that is essentially what I was getting at, we will need to revisit the other database images. Would you mind creating a tech debt task for this?

Lets make sure it all works with one application & its database before applying the changes elsewhere.

Reply:
👍 That sounds sensible to me. Lets make sure that these changes work and if so, I'll create tasks to revisit BAPI & ISAPI's db images to apply these changes.